samples/test-vulnerable-20260130.py

```
@@ -0,0 +1,57 @@
+import pickle
+
+# ========== SAST 취약점 (Semgrep) ==========
+
+# 1. Command Injection (CRITICAL)
+def run_command(user_input):
+    os.system(f"echo {user_input}")  # shell injection
+    subprocess.call(user_input, shell=True)  # command injection
+
+# 2. SQL Injection (HIGH)
+def get_user(user_id):
+    query = "SELECT * FROM users WHERE id = '" + user_id + "'"  # sqli
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)